website/docs/1_concepts/1_introduction.md

```diff
@@ -20,10 +20,7 @@ Some of the high-level capabilities and objectives of Apache Cassandra include,
 - Manage operations on pods through CassKop plugin (cleanup, rebuild, upgradesstable, removenode..)
 - Performing live Cassandra repairs through the use of [Cassandra reaper](http://cassandra-reaper.io/)
 - Multi-site management through [Multi-Casskop operator](https://github.com/Orange-OpenSource/casskop/tree/master/multi-casskop)
-
-Some of the roadmap features :
-
-- Live Backup/Restore
+- Live Backup/Restore of Cassandra's datas
```

Comment:
Other features in addition ?

Reply:
Let's remove features cause there is no real roadmap anymore

Reply:
Done